spec/unit/mixin/openssl_helper_spec.rb

```diff
@@ -862,7 +862,7 @@
       @certfile = Tempfile.new("certfile")
     end
 
-    context "When the cert file doesnt not exist" do
+    context "When the cert file doesn't not exist" do
```

Comment:
This is a crazy confusing statement

Reply:
This is what happens when you have lots of changes to deal with. will fix it thanks!